File: 프로그래머스/lv1/12937. 짝수와 홀수/짝수와 홀수.java
class Solution {
    public String solution(int num) {
        String answer = "";
        if(num % 2 == 0) {
			answer = "Even";
		} // num이 짝수라면 Even을 answer에 넣어주고
		else {
			answer = "Odd";
		} // 홀수라면 Odd를 넣어준다
        return answer;
    }
}

File: 프로그래머스/1/12937. 짝수와 홀수/짝수와 홀수.js
function solution(num) {
    let answer = "";
    if(num % 2 === 0) {
        answer = "Even";  
        // num이 짝수라면 "Even"을 answer에 넣어준다
    } else {
        answer = "Odd";   
        // num이 홀수라면 "Odd"를 answer에 넣어준다
    }
    return answer;
}
